src/main/java/duke/Task.java

```diff
@@ -0,0 +1,23 @@
+package duke;
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
```

Comment:
Perhaps need a descriptive header comments for the Task method? 🤔
And I noticed the same issue in several other places below.